### ISSUE TITLE
Time and Date should be set on the serverside

### ISSUE THREAD

**Issue by Jamesdwi** [dwyl/hapi-socketio-redis-chat-example#82]
if time and date were calculated on the server side, messages would always appear in order, this becomes important if the users are in different timezones. Or misconfigured time settings on clients.

**Comment by nelsonic** [comment]
@Jamesdwi agreed. we had this discussion on https://github.com/dwyl/time/issues/196 for a similar reason. 👍 
If you have have time to Pull Request a change/update to `this` project we would be very grateful!
